.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
   "deno.enable": true,
   "deno.unstable": true,
+  "deno.lint": true,
   "editor.formatOnSave": true,
   "prettier.printWidth": 100,
```

Comment:
It might probably be for the best to remove anything prettier related too then if we're going to use dprint/deno fmt